src/applications/disability-benefits/686c-674/config/chapters/674/student-networth-information/studentNetworthInformation.js

```diff
@@ -0,0 +1,90 @@
+import { TASK_KEYS } from '../../../constants';
+import { isChapterFieldRequired } from '../../../helpers';
+import { StudentNameHeader } from '../helpers';
+
+export const schema = {
+  type: 'object',
+  properties: {
+    studentDoesHaveNetworth: {
+      type: 'boolean',
+    },
+    networthInformation: {
+      type: 'object',
+      properties: {
+        savings: {
+          type: 'string',
+          pattern: '^\\d{1,6}',
```

Comment:
...is it reasonable to think a student wouldn't have a networth greater than six digits, or should this be uncapped?